### DISCUSSION TITLE
Object Detection and Tracking with Ultralytics YOLOv8!

### DISCUSSION THREAD

**Discussion by NLadi** [ultralytics/ultralytics#11805] [Show and tell]
Our new blogpost by [Nicolai Nielsen](https://github.com/niconielsen32) showcases a walkthrough for object detection and tracking with Ultralytics YOLOv8. This guide will take you step by step through the process of detecting and tracking objects on video and live on webcam. 

🔎 Key Highlights:  
✅ Configuring the Tracker 
✅ Real-Time Inference 
✅ Live Webcam Testing
✅ Practical Applications Across Industries

Learn more ➡️ https://www.ultralytics.com/blog/object-detection-and-tracking-with-ultralytics-yolov8

Join our GitHub Discussions to share your thoughts and learn more from the community!

[![Episode 7 Object detection and tracking with Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics/assets/155453586/58475990-1264-48ce-b4d8-cc5dbeca9cf3)](https://www.ultralytics.com/blog/object-detection-and-tracking-with-ultralytics-yolov8)

**Comment by pderrenger** [comment]
@NLadi thank you for sharing this, Nicolai! This is a fantastic resource for anyone interested in diving deeper into object detection and tracking using Ultralytics YOLOv8. 🚀

For those working with the model or exploring this feature for the first time, following through the guide on the blog will be incredibly helpful. If you have any specific questions or hit any roadblocks while implementing these features, don't hesitate to ask in our GitHub Discussions. Our community is always here to help!

Happy tracking! 📹👀
